### PR TITLE
Release Godot 4.1.3 stable and publish release notes

### PR DESCRIPTION
**Pull request by YuriSizov**
To be released tomorrow before midday.

### CODE REVIEW

File: collections/_article/maintenance-release-godot-4-1-3.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+excerpt: "With stability in mind, Godot contributors offer yet another set of changes for Godot 4.1 users, addressing a variety of bugs related to rendering, input, GUI, and platform support."
+categories: ["release"]
+author: Yuri Sizov
+image: /storage/blog/covers/maintenance-release-godot-4-1-3.webp
+image_caption_title: Meowing Point
+image_caption_description: A game by yofrancisco
+date: 2023-11-02 11:00:00
+---
+
+After about a week of testing Godot 4.1.3 is ready for production use. As Godot 4.2 [is being finalized](/article/dev-snapshot-godot-4-2-beta-4) we stay committed to provide support for users remaining on the current stable version of the engine. Our architecture and approach to release management allow us to easily pick many bugfixes and smaller improvements from the upcoming release and apply them to previous releases without compromising their stability.
```
Comment (JohnVeness) [inline collections/_article/maintenance-release-godot-4-1-3.md]:
```suggestion
After about a week of testing, Godot 4.1.3 is ready for production use. As Godot 4.2 [is being finalized](/article/dev-snapshot-godot-4-2-beta-4) we stay committed to providing support for users remaining on the current stable version of the engine. Our architecture and approach to release management allow us to easily pick many bugfixes and smaller improvements from the upcoming release and apply them to previous releases without compromising their stability.
```

Comment (YuriSizov) [reply collections/_article/maintenance-release-godot-4-1-3.md]: You know, that's what I wrote initially but then thought it didn't sound right.

Comment (JohnVeness) [reply collections/_article/maintenance-release-godot-4-1-3.md]: The comma or the "providing"?

Comment (YuriSizov) [reply collections/_article/maintenance-release-godot-4-1-3.md]: The "providing".

Comment (JohnVeness) [reply collections/_article/maintenance-release-godot-4-1-3.md]: I *think* "providing" is correct. I'm considering other phrases, like if someone takes over a company, they might say "we're committed to running the company as before", not "we're committed to run the company as before".

Comment (YuriSizov) [reply collections/_article/maintenance-release-godot-4-1-3.md]: Well your suggestion matches my gut feeling, apparently. So I'll update it :)

File: _data/versions.yml [hunk 2]
```diff
@@ -32,15 +32,19 @@
       release_notes: "/article/dev-snapshot-godot-4-2-dev-1/"
 
 - name: "4.1.3"
-  flavor: "rc1"
-  release_date: "27 October 2023"
-  release_notes: "/article/release-candidate-godot-4-1-3-rc-1/"
+  flavor: "stable"
+  release_date: "1 November 2023"
```
Comment (akien-mga) [inline _data/versions.yml:36]:
```suggestion
  release_date: "2 November 2023"
```

**Review by akien-mga — APPROVED** [review]

File: _data/versions.yml (hunk 2 above)
Comment (YuriSizov) [reply _data/versions.yml:36]: I used the release date when binaries were published, not the date of the article :P

**Comment by YuriSizov** [comment]
Thanks for reviews!

**Pull request merged**